lib/sass.dart

```diff
@@ -22,13 +22,27 @@ import 'src/visitor/serialize.dart';
 /// [SyncPackageResolver]: https://www.dartdocs.org/documentation/package_resolver/latest/package_resolver/SyncPackageResolver-class.html
 ///
 /// Finally throws a [SassException] if conversion fails.
-String render(String path,
-    {bool color: false, SyncPackageResolver packageResolver}) {
-  var contents = readFile(path);
-  var url = p.toUri(path);
-  var sassTree = p.extension(path) == '.sass'
-      ? new Stylesheet.parseSass(contents, url: url, color: color)
-      : new Stylesheet.parseScss(contents, url: url, color: color);
+String render(
+    {String path,
+    String data,
+    bool color: false,
+    SyncPackageResolver packageResolver}) {
```

Comment:
I don't think I want to change the signature here. The existing `render()` signature is clean and useful. I'd much rather add a separate function that takes `data`.